src/reducers/reducer.js

```diff
@@ -747,10 +747,7 @@ const onSetUnleashClient = (state, { payload }) => ({
 
 const onSetServerInfo = (state, action) => ({
   ...state,
-  serverInfo: {
```

Comment:
We should update the default redux value for `serverInfo` and the docstring

Reply:
Done, thanks